pkg/recordset/recordset.go

```diff
@@ -178,6 +213,29 @@ func getStacks(cl client.StackDescribeLister, re *regexp.Regexp, installation st
 	return result, nil
 }
 
+func filterStacksByStatus(input []cloudformation.Stack, statuses []string, exclude bool) []cloudformation.Stack {
```

Comment:
That thing looks like it deserves some unit tests.

Reply:
added unit tests, PTAL